### PR TITLE
fix: navigation bug 🔨🔨

### PR DESCRIPTION
**Pull request by hamza94max**

### CODE REVIEW

**Comment by MahmoudMabrok** [comment]
please change name of repo to reflect that it fix not a UI. 

also, please add description to PR itself.

File: app/src/main/java/com/islamey/hamza/wazaker/ui/HomeFragment/HomeFragment.kt [hunk 1]
```diff
@@ -41,7 +42,9 @@ class HomeFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        hijriViewModel.getHijriDate(getCurrentDate())
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            hijriViewModel.getHijriDate(getCurrentDate())
+        }
```
Comment (MahmoudMabrok) [inline app/src/main/java/com/islamey/hamza/wazaker/ui/HomeFragment/HomeFragment.kt:47]: could we add a comment with reason to do this.

Comment (MahmoudMabrok) [inline app/src/main/java/com/islamey/hamza/wazaker/ui/HomeFragment/HomeFragment.kt:47]: could we add a comment with reason to do this.

**Review by MahmoudMabrok — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/islamey/hamza/wazaker/ui/HomeFragment/HomeFragment.kt (hunk 1 above)
Comment (hamza94max) [reply app/src/main/java/com/islamey/hamza/wazaker/ui/HomeFragment/HomeFragment.kt:47]: as this function is not working with SDK below O

Comment (MahmoudMabrok) [reply app/src/main/java/com/islamey/hamza/wazaker/ui/HomeFragment/HomeFragment.kt:47]: share me a link for that.

Comment (hamza94max) [reply app/src/main/java/com/islamey/hamza/wazaker/ui/HomeFragment/HomeFragment.kt:47]: [stackOverFlow link](https://stackoverflow.com/questions/47480732/what-is-the-purpose-of-the-condition-if-build-version-sdk-int-build-version/51381567#51381567)

**Pull request merged**